tpl/.drone.yml.gotmpl

```diff
@@ -22,6 +22,13 @@ steps:
 {{- if eq .provider "slack" }}
   - name: slack-start
     image: plugins/slack
+    resources:
+      limits:
+        cpu: 200
```

Comment:
Is e.g. `cpu: 200` valid quantity syntax? (https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes) 

I can't find anything about it in the documentation and never have seen it before (it seems like you allocate 200 cpu).

Reply:
it's how it is in the drone docs...tested and works